Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -189,11 +190,14 @@ void CSequenceBatchRenderDialog::OnInitDialog()
 
     // Fill the sequence combo box.
     bool activeSequenceWasSet = false;
-    if (GetIEditor()->GetMovieSystem())
+    IMovieSystem* movieSystem = nullptr;
+    Maestro::MovieSystemRequestBus::BroadcastResult(movieSystem, &Maestro::MovieSystemRequestBus::Events::GetMovieSystem);
```

Comment:
As it leads to a lot of EBus calls I wonder if perf-wise we would want to cache the result in the constructor, or if it not worth it

Reply:
This is not a time critical operation, only happens during the dialog initialization, at a glance it may be better than what was already there which was doing a GetIEditor and GetMovieSystem function call for each sequence, and once in the for loop.

Reply:
One option is to use AZ::Interface if the movie system is a singelton